src/BuildAfterTargetingPack/BuildAfterTargetingPack.csproj

```diff
@@ -52,13 +52,13 @@
     </MSBuild>
   </Target>
 
-  <Target Name="CleanDelayedProjects" BeforeTargets="Clean" Condition=" '$(DotNetBuildFromSource)' != 'true' ">
+  <Target Name="CleanDelayedProjects" BeforeTargets="Clean" Condition=" '$(DotNetBuild)' != 'true' ">
     <MSBuild Projects="@(RequiresDelayedBuild)" BuildInParallel="$(BuildInParallel)" Targets="Clean" />
   </Target>
 
   <Target Name="CreateHelixPayloadDelayedProjects"
       BeforeTargets="CreateHelixPayload"
-      Condition=" '$(DotNetBuildFromSource)' != 'true' "
+      Condition=" '$(DotNetBuild)' != 'true' "
```

Comment:
This should probably be conditionalized on -test?

Reply:
Not necessarily, the regular build step creates the helix payload, then we send the payloads to helix in the following step: https://github.com/dotnet/aspnetcore/blob/983caa6afb09aeb82952bec077a59ab79514a288/.azure/pipelines/ci.yml#L740-L746